beta-src/src/components/ui/WDUnitController.tsx

```diff
@@ -50,6 +50,18 @@ const WDUnitController: React.FC<UnitControllerProps> = function ({
         }),
       );
     },
+    DISBAND: (command) => {
+      const [key] = command;
+      console.log("key", key);
+      setIconState(UIState.DISBANDED);
+      dispatch(
+        gameApiSliceActions.deleteCommand({
+          type: "unitCommands",
+          id: meta.unit.id,
+          command: key,
+        }),
+      );
```

Comment:
Create a `deleteCommand` function in this component similar to how it is done in WDTerritory.tsx now that this component will have more than one command.